src/components/StoryList.component.js

```diff
@@ -38,8 +38,14 @@ class StoryList extends Component {
   }
 
   onClickDelete(id) {
-    //<--TODO: Add delete message
-    console.log('Delete Button #' + id)
+    const { selectedEvent, loadStories } = this.props
+    if (
+      window.confirm(
```

Comment:
Using `window.confirm` is okay for now, but we might want something prettier (like a modal) later